neuraxle/base.py

```diff
@@ -823,18 +1256,50 @@ def setup(self) -> BaseStep:
         return self
 
     def set_train(self, is_train: bool=True):
+        """
+        Set pipeline step mode to train or test. Also set wrapped step mode to train or test.
```

Comment:
We could probably just say "This method overrides the method of BaseStep to also consider the wrapped step as well as self. See also: :func BaseStep.set_train"

This would avoid duplicate docstrings.